pkgs/applications/emulators/duckstation/default.nix

```diff
@@ -92,10 +91,9 @@ stdenv.mkDerivation rec {
     runHook postCheck
   '';
 
-  # Libpulseaudio fixes https://github.com/NixOS/nixpkgs/issues/171173
   qtWrapperArgs = [
     "--set QT_QPA_PLATFORM xcb"
-    "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libpulseaudio vulkan-loader ]}"
```

Comment:
This isn't needed anymore, since the SDL version of DuckStation (which is what required this) isn't a thing anymore (I should have removed it earlier).